### DISCUSSION TITLE
V2 and backward compatibility plans?

### DISCUSSION THREAD

**Discussion by JulienVincenot** [acids-ircam/RAVE#175] [General]
Hello,

I'm just curious since now v2 can be used by everyone thanks to [moiseshorta](https://github.com/moiseshorta) ( #162 )
is there anything general we should now about it beyond that it's improved / faster / higher quality?

One little worry I have is compatibility of models trained with v1 when v2 or later will become the norm. I'm personally still using v1 because I'm relying on the prior for some current project.

It seems nn~ versioning is separate from rave's versioning so I'm not too worried, but I'm a mac user without a serious GPU so I'm stuck with losing money to Colab for a while. I really do hope those models won't become unusable anytime soon... 😅

Thanks for you thoughts !

Julien

**Comment by caillonantoine** [comment]
Hi :) One of the most impotant changes in the v2 is the overall configurability of the model. You can choose between different model types (architectures / regularization) according to your project.

You are right about nn~, it is a completely different project, and previous versions of RAVE will stay compatible with it ! So don't worry about your previous trainings, they are still valid :)

**Reply by JulienVincenot** [reply]
ok great to know ! looking forward to play more with v2 when I'm done with that first project, merci Antoine !
